Cargo.toml

```diff
@@ -1,9 +1,10 @@
 [workspace]
 
 members = [
-  "propolis",
   "cli",
   "client",
+  "package",
```

Comment:
I request that we exclude this from the `default-members` of the workspace, so it's not built by default when iterating on propolis on a dev machine.

Reply:
Done - added a `default-members` to the workspace `Cargo.toml`.

Tested by inserting some junk characters in `package/src/main.rs`:
- `cargo build` still compiles fine
- `cargo run -p propolis-package` hits the (expected) compiler error

Reply:
Thanks!